service/server.py

```diff
@@ -25,6 +25,8 @@
 POSTCODE_REGEX = re.compile(address_utils.BASIC_POSTCODE_REGEX)
 NOF_SECS_BETWEEN_LOGINS = 1
 LOGGER = logging.getLogger(__name__)
+INCLUDE_TIME_IN_DATE_FORMAT = True
+IGNORE_TIME_IN_DATE_FORMAT = False
```

Comment:
Sorry, I didn't give these two enough thought. Do we need these constants? Isn't it that whenever we format date, we always indicate whether we want date with time component? Can't see a benefit over just using true/false. I may be missing something.